runtime/pom.xml

```diff
@@ -69,6 +68,15 @@
     </build>
 
     <profiles>
+        <profile>
+            <id>jdk9FF</id>
+            <activation>
+                <jdk>(1.8</jdk>
```

Comment:
Is this legal syntax? It seems to work, but I don't see anything in the docs about this pattern: https://maven.apache.org/enforcer/enforcer-rules/versionRanges.html

It looks to me like the standard way to say "greater than 1.8" would be `(1.8,)`.